common/src/main/java/com/revenuecat/purchases/common/HTTPClient.kt

```diff
@@ -101,6 +101,16 @@ class HTTPClient(
         requestHeaders: Map<String, String>,
         refreshETag: Boolean = false
     ): HTTPResult {
+        if (appConfig.forceServerErrors) {
```

Comment:
In iOS, we additionally have a compile time flag so this is only available for debug builds. It's possible to split that up in Android by having different versions in debug/release folders but not sure it's worth it...